core/modules/wmgr/python/dbMgr.py

```diff
@@ -810,7 +944,7 @@ def getIndex(dbName, tblName, chunkId=None):
             if not rows:
                 break
             for row in rows:
-                result.append(list(row))
+                result.append(tuple(row))
 
```

Comment:
Here, I'd like to have an additional documentation page which summarize data-loading architecture, where does related services run, which protocols are used, who authenticate where, and a hogh-level description of the workflow. I think it could be done shortly with some well-documentend figures?